packages/webapp/src/pages/election/round-video-upload.tsx

```diff
@@ -0,0 +1,348 @@
+import React, { useEffect, useState } from "react";
+import { RiVideoUploadLine } from "react-icons/ri";
+import { Dayjs } from "dayjs";
+
+import {
+    FluidLayout,
+    isValidDelegate,
+    onError,
+    uploadIpfsFileWithTransaction,
+    uploadToIpfs,
+    useCommunityGlobals,
+    useCurrentElection,
+    useLocalStorage,
+    useOngoingElectionData,
+    useUALAccount,
+} from "_app";
+import {
+    Container,
+    Text,
+    Expander,
+    Heading,
+    Loader,
+    VideoSubmissionFormAndPreview,
+    VideoSubmissionPhase,
+    Button,
+    Link,
+} from "_app/ui";
+import {
+    CurrentElection,
+    ElectionStatus,
+    ErrorLoadingElection,
+    setElectionRoundVideo,
+    getRoundTimes,
+} from "elections";
+import { EncryptionPasswordAlert } from "encryption";
+import { EdenMember } from "members";
+import { RoundHeader } from "elections/components/ongoing-election-components";
+
+// TODO:
+// 1) add buttons that launch video upload service in new tab
+// 2) test inductions to ensure I haven't screwed up their video upload
+// 3) Test video uploads during (and after) an election
+
+export const RoundVideoUploadPage = () => {
+    const [isElectionComplete, setIsElectionComplete] = useState(false);
+    const {
+        data: currentElection,
+        isLoading: isLoadingElection,
+        isError: isErrorElection,
+    } = useCurrentElection();
+
+    useEffect(() => {
+        if (currentElection?.electionState === ElectionStatus.Registration) {
+            setIsElectionComplete(true);
+        }
+    }, [currentElection]);
+
+    const isLoading = isLoadingElection;
+    const isError = isErrorElection;
+
+    if (isLoading || !currentElection) return <LoaderSection />;
+    if (isError) return <ErrorLoadingElection />;
+
+    const renderBanner = true; // TODO: get end time for video submission
+
+    return (
+        <FluidLayout
+            title="Election-video upload service"
+            banner={
+                renderBanner && (
+                    <EncryptionPasswordAlert
+                        promptSetupEncryptionKey={
+                            currentElection?.electionState !==
+                            ElectionStatus.Registration
+                        }
+                    />
+                )
+            }
+        >
```

Comment:
suggestion (non-blocking): We should use our new `SideNavLayout` here. (in this PR or a fast follow). And do we need the Encryption Password Alert to show up on this page? I don't think the password is used for video upload, is it?

Reply:
its not. removing the banner and adjusting the sidenav layout